README.md

```diff
@@ -1,61 +1,93 @@
-Dogtag PKI
-==========
+# Dogtag PKI
 
-[![Build Status](https://travis-ci.org/dogtagpki/pki-nightly-test.svg?branch=master)](https://travis-ci.org/dogtagpki/pki-nightly-test)
+![Required Tests](https://github.com/dogtagpki/pki/workflows/Required%20Tests/badge.svg)
+![QE Tests](https://github.com/dogtagpki/pki/workflows/QE%20Tests/badge.svg)
 
-(C) 2008 Red Hat, Inc.
-All rights reserved.
+The Dogtag Certificate System is an enterprise-class open source Certificate Authority (CA). It is a full-featured system, and has been hardened by real-world deployments. It supports all aspects of certificate lifecycle management, including key archival, OCSP and smartcard management, and much more.
 
-This Certificate System is open-source software.
+There are 5 different subsystems included in the Dogtag PKI suite:
 
-Please comply with the LICENSE contained in each of
-the individual components, and the EXPORT CONTROL
-regulations defined at:
+1. [Certificate Authority (CA) subsystem](https://www.dogtagpki.org/wiki/Certificate_Authority)
+2. [Key Recovery Authority (KRA) subsystem](https://www.dogtagpki.org/wiki/Key_Recovery_Authority)
+3. [Online Certificate Status Protocol (OCSP) subsystem](https://www.dogtagpki.org/wiki/OCSP_Manager)
+4. [Token Key Service (TKS) subsystem](https://www.dogtagpki.org/wiki/Token_Key_Service)
+5. [Token Processing System (TPS) subsystem](https://www.dogtagpki.org/wiki/Token_Processing_System)
 
-http://www.dogtagpki.org/wiki/PKI_Download
+## Documentation
 
-## Content
+The best place to start learning about the product is the [Dogtag PKI Wiki](https://www.dogtagpki.org)
 
-These directories contain the following:
+## Installing
 
-* base
+### Fedora/CentOS/RHEL
 
-  Contains most of the base source code
-  needed to build this project.  Note that
-  this directory does NOT contain
-  implementation specific user-interface
-  components required to build a working
-  Certificate System.
+To install the **whole Dogtag PKI suite**:
 
-* cmake
+````bash
+sudo dnf install dogtag-pki
```

Comment:
Actually, I don't think this works on RHEL, and I'm not even sure it works on RHCS with a LP entitlement. I don't think we do a `Provides: dogtag-pki`; it'd have to be `redhat-pki`, but we also don't do a `Provides: redhat-pki` on Fedora.

So perhaps leave these as Fedora-only instructions? :)

Reply:
Good catch. Let me fix it!